packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -14,6 +19,19 @@ const cli = {
         alias: 'config',
         describe: 'A file path to the test runner configuration.',
       },
+      disableSeleniumService: {
+        type: 'boolean',
+        describe: 'A flag to disable the selenium docker service.',
+        default: false,
+      },
```

Comment:
I think this is probably fine to keep, but I don't know of any use-cases anymore after updating travis to run against the docker service. The service is automatically disabled if a gridUrl is provided. If you're not running against the grid, you need to run against the selenium docker service.